### DISCUSSION TITLE
I need help with ublock. How do I get it, this is too complicated for me to get a response...I've no ideahow to get help.

### DISCUSSION THREAD

**Discussion by security-detection** [uBlockOrigin/uBlock-issues#3459] [General]
I cannot log into support...won't work. bitdefender keeps blocking gitcdn.xyz. How do I ask a question, get help for it?

**Comment by stephenhawk8054** [comment]
This is not bitdefender support forum.

**Comment by security-detection** [comment]
You offer no help. I needed help understanding issues with ublock accessing gitcdn and causing bitdefender to block it. It is ublock that had the issue. After attempting, multiple time to get help, which was refused, login failed to work...not a bug, not a support issue, unable to find ANY user info, I came here for help. I found a tutorial on youtube and removed it from ublock. The attitude and lack of help from UBlock is offensive. Thanks for nothing!!!!!!!!!!!!!!!!!!!

**Reply by stephenhawk8054** [reply]
Can you click on uBO icon > 💬 [Report button](https://github.com/gorhill/uBlock/wiki/Quick-guide:-popup-user-interface#the-tools) > scroll to the bottom > click on `Troubleshooting Information` and copy paste it here?

**Reply by garry-ut99** [reply]
> security-detection : ublock accessing gitcdn and causing bitdefender to block it. It is ublock that had the issue.

In the past yes:
- https://www.reddit.com/r/uBlockOrigin/comments/rdy62f/bitdefender_detects_ublock_origin_as_a_virus/
- https://github.com/uBlockOrigin/uBlock-issues/issues/1533
- https://github.com/uBlockOrigin/uBlock-issues/issues/1544

but not anymore, as currently uBO does not use / access anymore neither `gitcdn.xyz` nor `gitcdn.link`:
- https://www.reddit.com/r/uBlockOrigin/comments/1ftnbaf/firefox_ublock_origin_gitcdnxyz_url_blocked_by/
- https://github.com/gorhill/uBlock/commit/60b9aa2f38deaf80319518b3848437c6f6ea527c + https://github.com/gorhill/uBlock/commit/bd1e82418b2cc3b2fb25851ffc518170fd66af33 (removal of `gitcdn.xyz`)
- https://github.com/gorhill/uBlock/commit/8f66de67e17e5c0c674570701cd9387219cf34fe + https://github.com/gorhill/uBlock/commit/f80c84ab40b7a2a6d436e6ce3f72d11eea7303c4 (removal of `gitcdn.link`)

so most likely caused by your own custom manually added filters / filterlists, unless it's triggered by a string in a filter:
(but in such a case this is still rather an issue with BitDefender than with uBlock):
- https://github.com/uBlockOrigin/uAssets/blob/5da2c373d32f524e72428f2fe74ee331f92f45b7/filters/filters-2024.txt#L460
(by the way there exist also (`.link` only): 
https://github.com/uBlockOrigin/uAssets/blob/5da2c373d32f524e72428f2fe74ee331f92f45b7/filters/filters-2020.txt#L1050)
